engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -26,6 +26,9 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
+    public static final int ENABLED = GL_LINE;
+    public static final int DISABLED = GL_FILL;
+
```

Comment:
So, I'd leave those static variables but I'd make them store true and false respectively instead of GL constants.